students.md

```diff
@@ -10,3 +10,4 @@
 | Keenan | [@kmanpearl](https://github.com/kmanpearl) | [@wayscience] | Breakfast | 
 | Ethan | [@Ecohen1763](https://github.com/Ecohen1763) | Way lab | Sushi |
 | Dayna | [@olsonday](https://github.com/olsonday) | Way Lab | Soup |
+| Alex | [@alexKotz-koz] (https://github.com/alexKotz-koz) | Way Lab | Bagel |
```

Comment:
```suggestion
| Alex | [@alexKotz-koz](https://github.com/alexKotz-koz) | Way Lab | Bagel |
```

Removing the space to fix the markdown url reference. Otherwise, looks good!